db/migrate/20210927092807_migrate_stripe_account_owner_to_stripe_sca.rb

```diff
@@ -0,0 +1,17 @@
+class MigrateStripeAccountOwnerToStripeSca < ActiveRecord::Migration[6.1]
+  def up
+    replace_preferences_key("/spree/gateway/strip_connect", "/spree/gateway/stripe_sca")
+  end
+
+  private
+
+  def replace_preferences_key(from_pattern, to_pattern)
+    updated_pref_key = "replace( pref.key, '" + from_pattern + "', '" + to_pattern + "')"
+    Spree::Preference.connection.execute(
```

Comment:
I think that you don't need to reference the model here:
```suggestion
    execute(
```